server/app.js

```diff
@@ -33,6 +29,10 @@ if (process.env.NODE_ENV === 'production') {
   app.use(enforceHTTPS());
 }
 
+function isMeetupRequest(req) {
```

Comment:
I don't like this- regular function definitions will get hoisted to the top, right?